tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "declaration": true,
     "outDir": "./dist",
     "strict": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "resolveJsonModule": true
```

Comment:
Won't adding `resolveJsonModule` and pulling in the package.json cause the `dist` structure to change?

Reply:
True, my bad.
FWIW, the following would work (i.e. not change the `dist` structure, but still give type information), but it's gross so I'd only do this if you're ok with it:
```ts
const packageJSON: typeof import('../package.json') = require('../package.json');
```
Otherwise, I'll just change it back to `require` without the annotation.

Reply:
Type annotation is fine thanks 👍

Reply:
I was actually wrong; that also changed the dist structure. Removed the compiler setting.